src/main/java/core/basesyntax/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
```

Comment:
The package structure should be more granular. Consider creating a `service` package and a `service.impl` package to separate interfaces from their implementations.